app/src/main/java/org/simple/clinic/search/PatientSearchScreen.kt

```diff
@@ -146,4 +153,39 @@ class PatientSearchScreen(context: Context, attrs: AttributeSet) : RelativeLayou
       else -> View.GONE
     }
   }
+
+  fun setEmptyFullNameErrorVisible(visible: Boolean) {
+    fullNameInputLayout.error = if (visible) {
+      resources.getString(R.string.patientsearch_error_empty_fullname)
+    } else {
+      null
+    }
+  }
+
+  fun setEmptyDateOfBirthAndAgeErrorVisible(visible: Boolean) {
```

Comment:
I feel there's a lot of common functionality between all the date of birth and age error functions. Can we extract this common stuff out?

Reply:
Done